src/ppx_const.ml

```diff
@@ -25,7 +25,7 @@ let const_eq x y = match x, y with
       String.equal x y
   | Pconst_char x, Pconst_char y ->
       Char.equal x y
-  | Pconst_string (x, _delim1), Pconst_string (y, _delim2) ->
+  | Pconst_string (x, _, _delim1), Pconst_string (y, _, _delim2) ->
```

Comment:
Can you confirm that this is only necessary for ppxlib 0.9.0->0.18.0 compatibility, and won't break compatibility with older versions of ocaml?

Reply:
Indeed, ppxlib will take care of ensuring compat with older OCaml versions + you gain 4.11 compatibility here.